systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/IKafkaClient.java

```diff
@@ -5,13 +5,12 @@
 package io.strimzi.systemtest.kafkaclients;
 
 import java.io.IOException;
-import java.security.KeyStoreException;
 
 public interface IKafkaClient<T> {
 
-    T sendMessages(String topicName, String namespace, String clusterName, int messageCount, long timeoutMs) throws InterruptedException, KeyStoreException, IOException;
-    T sendMessagesTls(String topicName, String namespace, String clusterName, String kafkaUsername, int messageCount, String securityProtocol, long timeoutMs) throws InterruptedException, KeyStoreException, IOException;
+    T sendMessages(String topicName, String namespace, String clusterName, int messageCount, long timeoutMs) throws RuntimeException, IOException;
```

Comment:
You don't declare RuntimeExceptions, that's the whole beauty of them.
Simply:
```
T sendMessages(String topicName, String namespace, String clusterName, int messageCount, long timeoutMs) throws IOException;
```
will suffice.

Reply:
Done and thanks for the catch.